wgsl/grammar/setup.py

```diff
@@ -0,0 +1,61 @@
+from os.path import isdir, join
```

Comment:
What invokes this script, and when?
What is this supposed to do?

Reply:
@dneto0 As we discussed during the editors' call, this file is autogenerated for build process with only changing the parts that need correction such as including scanner, which is necessary for checking the spec.